project/db/postgresql.conf

```diff
@@ -327,8 +327,8 @@ effective_cache_size = {{ salt['pillar.get']('postgresql_config:effective_cache_
                     # 0 disables.
 
 # These are relevant when logging to syslog:
-#syslog_facility = 'LOCAL0'
-#syslog_ident = 'postgres'
+syslog_facility = 'LOCAL0'
+syslog_ident = 'postgres'
```

Comment:
How about dropping a config file in `/etc/rsyslog.d` that'll route these messages to their own log file?